src/VisualStudio/VisualBasic/Impl/Microsoft.VisualStudio.LanguageServices.VisualBasic.vbproj

```diff
@@ -39,6 +39,8 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System.Windows.Forms" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.16.9.DesignTime" Version="$(MicrosoftVisualStudioShellInterop169DesignTimeVersion)" />
```

Comment:
Why is this necessary? The reference should have propagated.

Reply:
I was expecting that as well initially. Things do work OK during build without this.
After hitting the FileNotFoundException at runtime and trying several things, it occurred to me that this may be the issue.
I think now that the embedding itself is not transitive and so we need it in both the concrete implementations as well.

Reply:
I guess the EmbedInteropTypes might be running into the issue where MSBuild targets indeed have funky propagation rules; so keeping _that_ but not having the line 42 should be good. Is that easy to test?